blueprint/src/content.tex

```diff
@@ -215,9 +306,19 @@ \section{MainTheorem}
 \begin{lemma}\label{mul_expand₀}
   \lean{mul_expand₀}
   \uses{Finset.powerset₀,mul_expand₁}
+  \leanok
   The polynomial expansion of \(1 - \prod_{i=0}^{n-1} (1 - g(i))\).
 \end{lemma}
 
+\begin{proof}
+  \leanok
+  We construct a function \( g' : \mathbb{N} \to \mathbb{Z} \). When x < n, g'(x) is defined as g(x), when x >= n ,g'(x) is defined ad 0.
+
+  Since we only use the x < n part of g(x) in the statement, it is obvious that we can change g to g' in the statement.
+
+  Now the statement is completely the same as mul_expand1. 
```

Comment:
Do not use mul_expand1 here. The underline has special meaning in latex. \ref{mul_expand₁} is recommended.